xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -797,6 +814,7 @@ static void DisplayReconfigured(CGDirectDisplayID display,
   m_nWidth = newWidth;
   m_nHeight = newHeight;
   m_glContext = context;
+  g_graphicsContext.SetFPS(m_refreshRate);
```

Comment:
@Memphiz this is tool late and leaves GraphicsContext in an inconsistent state. If you set an override to fps, you have to do this in before you fire OnResetDisplay event. 
Now videoSyncOsx receives the ResetDisplay event and gets incorrect FPS from graphicsContext.